src/index.html.js

```diff
@@ -24,9 +24,13 @@ export default function renderDocument({ helmet, rootHtml, buildManifest }) {
                 ${assets.secondary.map((asset) => `<link rel="prefetch" href="${asset.url}">`).join('\n')}
             </head>
             <body>
-                <!-- Root element where app goes -->
-                <div id="root">${rootHtml}</div>
+                <div id="root">`;
+}
+
+export function renderRemainingBlock(buildManifest) {
+    const { assets } = buildManifest;
 
+    return `</div>
```

Comment:
Could we keep the same indentation of the opening div?

Reply:
i tried but i had this error: `Warning: Did not expect server HTML to contain the text node " " in <div>.`

Reply:
Does this happen if you still do that on a new line? e.g.:

```
 return `
    </div>
```

Reply:
yes. check this [issue](https://github.com/facebook/react/issues/10879).